test/main/modules.js

```diff
@@ -63,6 +87,45 @@ QUnit.test( "runs after final test", function( assert ) {
 	assert.expect( 1 );
 } );
 
+QUnit.module( "after (skip)", {
+	after: function( assert ) {
+		assert.ok( true, "after hook ran" );
+	}
+} );
+
+QUnit.test( "does not run after initial tests", function( assert ) {
+	assert.expect( 0 );
+} );
+
+QUnit.test( "runs after final unskipped test", function( assert ) {
+	assert.expect( 1 );
+} );
+
+QUnit.skip( "last test in module is skipped" );
+
+QUnit.module( "before/after with all tests skipped (wrapper)", function() {
```

Comment:
👍 -- let me know your decision on the other two comments, and I'll roll all three into one commit.